extensions/workramp/webhooks/EventWebhook.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  DataPointDefinition,
+  type Webhook,
+} from '@awell-health/extensions-core'
+import { WORKRAMP_IDENTIFIER } from '../config'
+import { settings } from '../settings'
+import { EventPayload, EventPayloadSchema } from '../types'
+
+const dataPoints = {
+  eventType: {
+    key: 'eventType',
+    valueType: 'string',
+  },
+} satisfies Record<string, DataPointDefinition>
+
+export const eventWebhook: Webhook<
+  keyof typeof dataPoints,
+  EventPayload,
+  typeof settings
+> = {
+  key: 'appointmentCreated',
+  dataPoints,
+  onWebhookReceived: async ({ payload }, onSuccess, onError) => {
+    const {
+      user: { id: value },
+      eventType,
+    } = EventPayloadSchema.parse(payload)
+    await onSuccess({
+      data_points: {
+        eventType: eventType.toString(),
+      },
```

Comment:
What if the care flow doesn't have this baseline info configured, would it still work or would it fail?

Reply:
it would still work. this isn't baseline info, it's a data point definition that is enabled by the webhook. i'm including it here because my assumption is there will be some conditional logic around it (e.g. if it's a completion path event)